workbench-core/example/infrastructure/cypress.config.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { defineConfig } from 'cypress';
+
+import { SSM } from '@aws-sdk/client-ssm';
+import { CognitoIdentityProvider } from '@aws-sdk/client-cognito-identity-provider';
+import fs from 'fs';
+import yaml from 'js-yaml';
+import { join } from 'path';
+import { merge } from 'lodash';
+
+const getSecret = async (ssm: SSM, name: string) => {
+  const response = await ssm.getParameter({
+    Name: name,
+    WithDecryption: true
+  });
+
+  return response.Parameter!.Value;
+};
+
+const getEnvironmentVariables = async (stage: string) => {
+  console.info('Stage:', stage);
+
+  const config = yaml.load(
+    fs.readFileSync(join(__dirname, `./integration-tests/config/${stage}.yaml`), 'utf8')
```

Comment:
Do we need this? 
We are basically requiring integ-test configuration in order to run e2e tests when IMO they should be independent of each other

Reply:
This is needed to retrieve the username and password of Cognito user in order to log in. The tests here are meant to be a complement to integration test suite that requires logging in with WebUI, not e2e UI tests per se. These tests are also ran from same Github action [deploy-integration-example-app.yml](https://github.com/aws-solutions/solution-spark-on-aws/pull/944/files#diff-8d34599e34eafa22976e842cd2c1438d89f2f7cce069e3f775f76eee3a028af8)

Reply:
got it, if they are mean to be coupled then its fine for me